bin/yaml/tools.yaml

```diff
@@ -172,3 +172,20 @@ tools:
     url: https://storage.googleapis.com/r8-releases/smali/{name}/{filename}
     targets:
       - 3.0.3
+  bloaty:
+    type: tarballs
+    compression: bz2
+    url: https://github.com/google/bloaty/releases/download/v{name}/bloaty-{name}.tar.bz2
+    dir: bloaty-{name}
+    check_exe: bin/bloaty --help
+    after_stage_script:
+      - mv bloaty-{{name}} bloaty-build
+      - mkdir -p bloaty-{{name}}/bin
+      - cd bloaty-build
+      - mkdir build
+      - cd build
+      - cmake ..
```

Comment:
Building this like this will mean it's tied to the architecture/configuration etc of the _installing_ machine (we sometimes have a different set of machines doing installation to a shared location, and may wish to update OSes etc separately). This might be OK for the short term (and indeed some of the other tools use this apoproach); but a longer term fix is to do a hermetic build (much more involved see [our misc builder](https://github.com/compiler-explorer/misc-builder)).

Reply:
Thanks for the feedback @mattgodbolt! I will definitely look at it and submit another PR.